server/plugin/command.go

```diff
@@ -299,8 +308,8 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		if len(optionList) > 1 {
 			return "Just one list of features is allowed"
 		} else if len(optionList) == 1 {
-			features = optionList[0]
-			fs := strings.Split(features, ",")
+			subscribeEvents.Features = optionList[0]
+			fs := strings.Split(subscribeEvents.Features, ",")
```

Comment:
Since we're making this its own type, we can abstract the `Split` functionality into its own method as well. Maybe something like `subscribeEvents.Split()` or `subscribeEvents.ToSlice()`